py/torch_tensorrt/dynamo/_defaults.py

```diff
@@ -6,6 +6,7 @@
 DEBUG = False
 DEVICE = None
 DISABLE_TF32 = False
+ASSUME_DYNAMIC_SHAPE_SUPPORT = False
```

Comment:
Are there cases where one might want this to be `True`?

Reply:
- Some of the ops might support dynamic shapes supports_dynamic_shapes=True might not be enabled (until we have validation).  
- The existing models which compiled successfully assumed all ops supported dynamic shapes but we have limited that now. This flag lists that restriction.